src/components/Dashboard.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div>
```

Comment:
Try to use some ident here. Having <div> and no identitation for inside <div>, it is hard to follow HTML structure. 
Do you agree?